.github/workflows/kind.yaml

```diff
@@ -0,0 +1,43 @@
+name: 'Kind setup'
```

Comment:
Note: this workflow will be moved to nightly. It is here just during code review to test functionality